src/shared/seccomp-util.c

```diff
@@ -977,18 +976,22 @@ int seccomp_load_syscall_filter_set_raw(uint32_t default_action, Hashmap* set, u
 
                 HASHMAP_FOREACH_KEY(val, id, set, i) {
                         uint32_t a = action;
-                        int e = PTR_TO_INT(val);
+                        int syscall = PTR_TO_INT(id) - 1;
```

Comment:
I think this should be renamed to e.g. `syscall_id` or something, as this 'conflicts' to `syscall()`.